index.html

```diff
@@ -7,12 +7,13 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap"
       rel="stylesheet"
-    />
```

Comment:
I'm always undecided on line break styles but the > looks a little off here. I'd keep it at the end of the attributes